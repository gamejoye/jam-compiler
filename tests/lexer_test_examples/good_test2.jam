Class Main extends B {
  let s: String = "Hello, gamejoye!";
  let x: (new B).f();
};

Class B {
  f(): Int {
    return 1;
  };
};