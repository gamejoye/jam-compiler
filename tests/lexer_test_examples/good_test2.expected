CLASS
TYPEID
EXTENDS
TYPEID
{
LET
OBJECTID
:
TYPEID
=
STR_CONST
;
LET
OBJECTID
:
(
NEW
TYPEID
)
.
OBJECTID
(
)
;
}
;
CLASS
TYPEID
{
OBJECTID
(
)
:
TYPEID
{
RETURN
INT_CONST
;
}
;
}
;